src/transport/SessionManager.cpp

```diff
@@ -656,6 +668,13 @@ void SessionManager::SecureUnicastMessageDispatch(const PacketHeader & packetHea
         return;
     }
 
+    // Drop secure unicast messages with privacy enabled.
+    if (packetHeader.HasPrivacyFlag())
+    {
+        ChipLogError(Inet, "Dropping secure unicast message with privacy flag set");
```

Comment:
So this PR is only supporting privacy decrypt for group messages?

Reply:
Yes.  I suppose we could add the processing to unicast as well -- it would only scramble the counters as src / dst are always elided.

Reply:
Per spec unicast messages generally have this, no?